augly/image/functional.py

```diff
@@ -1786,3 +1786,56 @@ def vflip(
     imutils.get_metadata(metadata=metadata, function_name="vflip", **func_kwargs)
 
     return imutils.ret_and_save_image(aug_image, output_path)
+
+
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    level: float = 5.0,
+    max_level: float = 10.0,
+    axis: str = "x_axis",
+    metadata: Optional[List[Dict[str, Any]]] = None,
+) -> Image.Image:
+    
+    """
+    Skew image with respect to X-axis or Y-axis
+    
+    @param image: the path to an image or a variable of type PIL.Image.Image
+        to be augmented
+        
+    @param level: Level of the operation that will be in [0, `PARAMETER_MAX`]
+    
+    @param max_level: Maximum value that the operation can have. This will be
+        scaled to level / PARAMETER_MAX.
+        
+    @param axis: Axis with respect to change like x-axis or y-axis
+                
+    @param output_path: the path in which the resulting image will be stored.
+        If None, the resulting PIL Image will still be returned
```

Comment:
```suggestion
    @param output_path: the path in which the resulting image will be stored.
        If None, the resulting PIL Image will still be returned

    @param level: Level of the operation that will be in [0, `PARAMETER_MAX`]
    
    @param max_level: Maximum value that the operation can have. This will be
        scaled to level / PARAMETER_MAX.
        
    @param axis: Axis with respect to change like x-axis or y-axis
```